src/sentry/seer/anomaly_detection/get_anomaly_data.py

```diff
@@ -153,3 +155,112 @@ def get_anomaly_data_from_seer(
         )
         return None
     return ts
+
+
+def get_anomaly_data_from_seer(
+    sensitivity,
+    seasonality,
+    threshold_type,
+    subscription: QuerySubscription,
+    subscription_update: MetricDetectorUpdate,
+    source_id,
+) -> list[TimeSeriesPoint] | None:
+    snuba_query: SnubaQuery = subscription.snuba_query
+    aggregation_value = subscription_update["values"]["value"]
+
+    extra_data = {
+        "subscription_id": subscription.id,
+        "organization_id": subscription.project.organization.id,
+        "project_id": subscription.project_id,
+        "source_id": source_id,
```

Comment:
Isn't `source_id` a dictionary? Do we want to log that or pull off the attributes?